tests/conftest.py

```diff
@@ -179,9 +179,9 @@ def build_test_backends_list(metafunc):
 
 def build_test_backends_list_cfe(metafunc):
     include_pandas: bool = True
-    include_spark: bool = not metafunc.config.getoption("--no-spark")
+    include_spark: bool = metafunc.config.getoption("--spark")
     include_sqlalchemy: bool = not metafunc.config.getoption("--no-sqlalchemy")
-    include_postgresql = not metafunc.config.getoption("--no-postgresql")
+    include_postgresql: bool = metafunc.config.getoption("--postgresql")
```

Comment:
After discussion with @donaldheppner 

* creation of 2 options. 
* `--postgresql` and `--no-postgresql`. And have them both be recognized. 
*  more of a "deprecation" approach rather than a deletion. 
* adding a warning : "this is going to be deprecated"